resources/QueryPopulateSSO_UsernameRunner.php

```diff
@@ -39,12 +39,10 @@
     ++$count;
     $cleanDN = cleanDN($user->getCertificateDn());
     if (!empty($cleanDN)) {
-        $url = "https://www.egi.eu/sso/api/user?dn=" . $cleanDN;
+        $url = "https://sso.egi.eu/admin/api/user?dn=" . $cleanDN;
         $ch = curl_init();
         curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
Not a problem introduced by this PR.

Maybe I'm misreading the curl manual, but for SSL_VERIFY_HOST -

![image](https://user-images.githubusercontent.com/20706670/124726539-c3ef4300-df05-11eb-9548-0c9eff79302a.png)

 it does imply to me that we are not checking that we trust the issuer CA of the server's certificate.

```suggestion
        curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
```

Reply:
Captured #333